package.json

```diff
@@ -54,33 +57,33 @@
   ],
   "dependencies": {
     "@typescript-eslint/parser": "^5.0.0",
-    "eslint-config-standard": "^16.0.0"
+    "eslint-config-standard": "^17.0.0-1"
```

Comment:
It is a prerelease, `17.0.0` doesn't exist yet:
Ref: https://github.com/standard/eslint-config-standard/issues/208

Reply:
This opens a discussion about prereleases. I'm not sure I'd like to go there.

Reply:
We don't need to do prereleases if you don't want to, and that's totally fine.
As said in the PR description:
> Should not be merged/released until `eslint-config-standard@17` has been released stable.

So the only thing that we'll need to change in this PR once released is this line to be: `"eslint-config-standard": "^17.0.0"`.

Reply:
Yes, defer merging until upstream release.

Reply:
@mightyiam There have been several requests to do a pre-release in the ESLint v8 issue. It appears the upstream will be in pre-release for a while. Can we please also get a pre-release of this package?

Reply:
This becomes a bit of a catch-22, as the `standard` prerelease is to allow projects to give feedback on regressions and/or missing features (like https://github.com/standard/standard/issues/1548)

Without a prerelease of this module, that excludes the users of `eslint-config-standard-with-typescript` from contributing to that.

Not sure why a prerelease would be a bad thing?

Reply:
Pre-release adds some complexity. A pre-release should be developed on a branch of its own, for one.

I am not aware of the value that users find in pre-releases. Would anyone care to explain?

Reply:
> A pre-release should be developed on a branch of its own, for one.

I would say that the main branch is where all development happen, no matter if it’s ending up in a stable release or a prerelease.

Separate branches as I see it is only used for maintaining legacy versions?

> I am not aware of the value that users find in pre-releases. Would anyone care to explain?

When we release a new major version we commit to a set of breaking changes. If those breaking changes doesn’t align with our intended breaking changes or the changes desired by the community, then we have to roll yet another major release to fix that.

Prereleases enables us + interested parties in the community to test out a new major before we commit to the breaking changes, making it possible for us to ensure that we ship what we intend to ship.

It also allows those in the community who feels blocked by current major version to unblock themselves earlier than would otherwise be possible.

Reply:
> > A pre-release should be developed on a branch of its own, for one.
> 
> I would say that the main branch is where all development happen, no matter if it’s ending up in a stable release or a prerelease.
> 
> Separate branches as I see it is only used for maintaining legacy versions?

Would you like the main branch to always be releasable?
> 
> > I am not aware of the value that users find in pre-releases. Would anyone care to explain?
> 
> When we release a new major version we commit to a set of breaking changes. If those breaking changes doesn’t align with our intended breaking changes or the changes desired by the community, then we have to roll yet another major release to fix that.
> 
> Prereleases enables us + interested parties in the community to test out a new major before we commit to the breaking changes, making it possible for us to ensure that we ship what we intend to ship.
> 
> It also allows those in the community who feels blocked by current major version to unblock themselves earlier than would otherwise be possible.

Thank you. This last paragraph I didn't understand. What is "current major version" in this scenario?

Reply:
`eslint-config-standard@17` has been released stable now. :tada: